sklearn/dummy.py

```diff
@@ -63,19 +64,19 @@ class DummyClassifier(MultiOutputMixin, ClassifierMixin, BaseEstimator):
 
     Attributes
     ----------
-    classes_ : array or list of array of shape (n_classes,)
+    classes_ : ndarray of shape (n_classes,) or list of ndarray
```

Comment:
I think a good description in this case would be `ndarray of shape (n_classes,) or list thereof`. This implicitly provides the shape of the arrays.

Jax does it like that https://jax.readthedocs.io/en/latest/jax.html#jax.vmap

It's not strictly in our guidelines though (not sure it needs to). Thoughts @glemaitre  ?

Reply:
I pretty like this solution.

Reply:
It looks neat and we don't have so many of these occurrences so yes I am not sure that we should add this in the User Guide.